lib/sqli/sqli_parser.y

```diff
@@ -236,6 +249,19 @@ expr:   func
             sqli_store_data(ctx, &$operator);
         }
         | expr post_exprs
+        | TOK_IF[tk] '('[u1] expr ','[u2] expr ')'[u3] {
```

Comment:
Some of the commit rules (and of the other commits) are duplicated. The original `expr` and `op_expr` rules also have duplicated rules. So, I've refactored that, see master branch. Please rewrite the commits to also remove code duplication.